controllers/alerts.js

```diff
@@ -1,22 +1,20 @@
 const express = require('express');
 const { Configuration } = require('@schul-cloud/commons');
+const asyncWrapper = require('../middleware/asyncMiddlewareWrapper');
 const api = require('../api');
 const logger = require('../helpers/logger');
 
 const router = express.Router();
 
-let handler = (req, res) => {
-	res.json([]);
-};
-if (Configuration.get('FEATURE_ALERTS_ENABLED')) {
-	handler = (req, res) => api(req).get('/alert')
-		.then((alert) => {
-			res.json(alert);
-		}).catch((err) => {
-			logger.error(new Error('Can not get /alert', err));
-			res.json([]);
-		});
-}
-router.get('/', handler);
+router.get('/', asyncWrapper(async (req, res, next) => {
```

Comment:
I prefer the old solution without wrapper and added catch with different handler for set feature flag and without. At the moment the wrapper is not needed at this place.